modular_zubbers/modules/code/alt_job_titles.dm

```diff
@@ -0,0 +1,421 @@
+/**
+ * This is the file you should use to add alternate titles for each job, just
+ * follow the way they're done here, it's easy enough and shouldn't take any
+ * time at all to add more or add some for a job that doesn't have any.
+ */
+
+/datum/job/ai
+	alt_titles = list(
+		"AI",
+		"Station Intelligence",
+		"Automated Overseer",
+		"Language Model",
+		"Mind Scan",
+		"Remote Helper Agent",
+	)
+
+/datum/job/assistant
+	alt_titles = list(
+		"Assistant",
+		"Civilian",
+		"Tourist",
+		"Businessman",
+		"Businesswoman",
+		"Trader",
+		"Entertainer",
+		"Freelancer",
+		"Artist",
+		"Off-Duty Staff",
+		"Off-Duty Crew",
+		"Collector",
+		"Maintenance Dweller",
+		"Hobbyist",
+		"Intern",
+		"Visitor",
+		"Gamer",
+	)
+
+/datum/job/atmospheric_technician
+	alt_titles = list(
+		"Atmospheric Technician",
+		"Life Support Technician",
+		"Emergency Fire Technician",
+		"Firefighter",
+		"HVAC Engineer",
+	)
+
+/datum/job/barber
+	alt_titles = list(
+		"Barber",
+		"Salon Manager",
+		"Salon Technician",
+		"Stylist",
+		"Colorist",
+	)
+
+/datum/job/bartender
+	alt_titles = list(
+		"Bartender",
+		"Mixologist",
+		"Barkeeper",
+		"Barista",
+	)
+
+/datum/job/blueshield
+	alt_titles = list(
+		"Blueshield",
+		"Command Bodyguard",
+		"Executive Protection Agent",
+	)
```

Comment:
These already exist, why's this redefined?

Reply:
Might need to prune my code for jobs that I haven't added alt titles to, I'll take care of it.